basic/src/test/java/org/jboss/hal/testsuite/test/runtime/elytron/CredentialStoreAliasTestCase.java

```diff
@@ -0,0 +1,127 @@
+            ModelNodeResult aliases = ops.invoke(READ_ALIASES_OPERATION, credentialStoreAddress);
+            aliases.assertSuccess();
+            Assert.assertTrue("Created alias should be present in model", aliases.stringListValue().contains(aliasName));
+        } finally {
+            ops.invoke(REMOVE_ALIAS_OPERATION, credentialStoreAddress, Values.of(ALIAS, aliasName));
+            adminOps.reloadIfRequired();
+        }
+    }
+
+    @Test
+    public void testAddAliasWithOptionalFields() throws IOException, TimeoutException, InterruptedException {
+        final String aliasName = "my_alias_" + RandomStringUtils.randomNumeric(7);
+        final String aliasSecretValue = RandomStringUtils.randomAlphanumeric(7);
+        try {
+            page.navigate();
+            page.getResourceManager().selectByName(CREDENTIAL_STORE_NAME);
+            ResourceManager aliasManager = page.getConfig()
+                    .switchTo(ALIASES_TAB_LABEL)
+                    .getResourceManager();
+            AddResourceWizard wizard = aliasManager.addResource(AddResourceWizard.class);
+            wizard.text(ALIAS, aliasName);
+            wizard.openOptionalFieldsTab();
+            wizard.text(SECRET_VALUE,aliasSecretValue)
+                    .saveAndDismissReloadRequiredWindowWithState().assertWindowClosed();
+            Assert.assertTrue(aliasManager.isResourcePresent(aliasName));
+            ModelNodeResult aliases = ops.invoke(READ_ALIASES_OPERATION, credentialStoreAddress);
+            aliases.assertSuccess();
+            Assert.assertTrue("Created alias should be present in model", aliases.stringListValue().contains(aliasName));
+        } finally {
+            ops.invoke(REMOVE_ALIAS_OPERATION, credentialStoreAddress, Values.of(ALIAS, aliasName));
+            adminOps.reloadIfRequired();
+        }
+    }
+
+    @Test
+    public void testRemoveAlias() throws IOException, TimeoutException, InterruptedException {
+        final String aliasName = "my_alias_" + RandomStringUtils.randomNumeric(7);
+        try {
+            ops.invoke(ADD_ALIAS_OPERATION,credentialStoreAddress,Values.of(ALIAS,aliasName)).assertSuccess();
+            page.navigate();
+            page.getResourceManager().selectByName(CREDENTIAL_STORE_NAME);
+            ResourceManager aliasManager = page.getConfig()
+                    .switchTo(ALIASES_TAB_LABEL)
+                    .getResourceManager();
+            aliasManager
+                    .removeResource(aliasName)
+                    .confirmAndDismissReloadRequiredMessage()
+                    .assertClosed();
+            Assert.assertFalse(aliasManager.isResourcePresent(aliasName));
+            ModelNodeResult aliases = ops.invoke(READ_ALIASES_OPERATION, credentialStoreAddress);
+            aliases.assertSuccess();
+            Assert.assertFalse("Created alias should be present in model", aliases.stringListValue().contains(aliasName));
```

Comment:
Please change the copy-pasted text `Created alias should be present in model` to something like `Removed alias should not be present in model anymore.`